gateway/api/ray.py

```diff
@@ -8,6 +8,10 @@
 import uuid
 from typing import Any, Optional
 
+import yaml
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
```

Comment:
out of curiosity, is there a reason to prefer the openshift dynamic client to the vanilla kubernetes one?

Reply:
The openshift one just adds "apply" function.  We don't use it so I'll change to the kubernetes one in the follow up PR.  Thanks for pointing it out.